packages/babel-helper-module-transforms/src/dynamic-import.ts

```diff
@@ -1,9 +1,67 @@
 // Heavily inspired by
 // https://github.com/airbnb/babel-plugin-dynamic-import-node/blob/master/src/utils.js
 
+import type { File } from "@babel/core";
 import * as t from "@babel/types";
 import template from "@babel/template";
 
+// TODO(Babel 8): Remove this
```

Comment:
We can wrap the fallback logic within `if (process.env.BABEL_8_BREAKING)` and get rid of yet another Babel 8 todo item. 😄